tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/Startup.cs

```diff
@@ -1,21 +0,0 @@
-﻿using Microsoft.Bot.Builder.Dialogs;
-using Microsoft.Bot.Builder.Dialogs.Adaptive;
-using Microsoft.Bot.Builder.Dialogs.Adaptive.Testing;
-using Microsoft.Bot.Builder.Dialogs.Declarative;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-namespace Microsoft.Bot.Builder
-{
-    [TestClass]
-    public class Startup
-    {
-        [AssemblyInitialize]
-        public static void Initialize(TestContext testContext)
-        {
-            ComponentRegistration.Add(new DeclarativeComponentRegistration());
-            ComponentRegistration.Add(new AdaptiveComponentRegistration());
-            ComponentRegistration.Add(new AdaptiveTestingComponentRegistration());
-            ComponentRegistration.Add(new LanguageGenerationComponentRegistration());
-        }
```

Comment:
This only runs once for ALL classes in the assembly.

Reply:
updated